cluster-scope/overlays/nerc-ocp-prod/feature/odf/redhatcop.redhat.io/odf-node-patcher.yaml

```diff
@@ -0,0 +1,200 @@
+---
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: Patch
+metadata:
+  name: odf-node-label-wrk-10
+  namespace: openshift-storage
+spec:
+  serviceAccountRef:
+    name: patcher
+  patches:
+    odf-node-label-wrk-10:
```

Comment:
Why are we only applying the `openshift-storage` label to a subset of the nodes on ocp-prod?

Reply:
The patch was for all worker nodes actually, after our discussion I have updated this to apply to all worker nodes by matching the Kubernetes worker label.